Tests/AsyncQueueTests/ActorQueueTests.swift

```diff
@@ -65,17 +65,17 @@ final class ActorQueueTests: XCTestCase {
 
     func test_enqueue_taskParameterIsAdoptedActor() async {
         let semaphore = Semaphore()
-        systemUnderTest.enqueue { counter in
-            XCTAssertTrue(counter === self.counter)
```

Comment:
capturing non-sendable `self` in the sendable closure is not good! So instead we now capture the `counter` directly.